<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MonitorSystem</class>
 <widget class="QMainWindow" name="MonitorSystem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>919</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Interactive Seam Tracking System</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_20">
    <item>
     <widget class="QLabel" name="originalImage">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>950</width>
        <height>740</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>基本功能</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="5,1,5">
           <item>
            <widget class="QPushButton" name="cameraOpen">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>打开</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="flipButton">
             <property name="text">
              <string/>
             </property>
             <property name="autoRepeatDelay">
              <number>100</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="cameraClose">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>关闭</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QPushButton" name="imageSave">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>保存</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="saveEnd">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>结束</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLineEdit" name="savePath"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>相机标定</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QPushButton" name="calibButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>标定</string>
             </property>
             <property name="iconSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="calibCapture">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>拍照</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>棋盘格：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="chessLength"/>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>mm</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QSpinBox" name="chessNumLength"/>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>×</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="chessNumWidth"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="fx">
             <property name="text">
              <string>fx:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="fy">
             <property name="text">
              <string>fy:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>ROI设定</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="2,0,0,0,0">
           <item>
            <widget class="QLabel" name="ROI_x_2">
             <property name="text">
              <string>ROI_x：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="ROI_x">
             <property name="maximum">
              <number>2000</number>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="ROI_y_2">
             <property name="text">
              <string>ROI_y：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="ROI_y">
             <property name="maximum">
              <number>2000</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="2,0,0,0,0">
           <item>
            <widget class="QLabel" name="ROI_w_2">
             <property name="text">
              <string>ROI_w：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="ROI_w">
             <property name="maximum">
              <number>2000</number>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="ROI_h_2">
             <property name="text">
              <string>ROI_h：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="ROI_h">
             <property name="maximum">
              <number>2000</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_13">
        <item>
         <widget class="QPushButton" name="add_click_button">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="undo_click_button">
          <property name="text">
           <string>Undo</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="clear_click_button">
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="model_roi_button">
          <property name="text">
           <string>roi</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_19">
          <item>
           <widget class="QPushButton" name="model_load_button">
            <property name="text">
             <string>Load Model</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="keyhole_click_button">
            <property name="text">
             <string>Keyhole</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>axis:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="trace_axis_box">
            <property name="editable">
             <bool>false</bool>
            </property>
            <property name="currentText">
             <string notr="true">x</string>
            </property>
            <item>
             <property name="text">
              <string>x</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>y</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_14">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_18">
          <item>
           <widget class="QLabel" name="label_25">
            <property name="text">
             <string>间隙：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="seamWidth"/>
          </item>
          <item>
           <widget class="QLabel" name="label_27">
            <property name="text">
             <string>mm</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>PTrace:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="trace_type_box">
            <item>
             <property name="text">
              <string>None</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Center</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Side</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_16">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_17">
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>偏差：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="seamDev"/>
          </item>
          <item>
           <widget class="QLabel" name="label_24">
            <property name="text">
             <string>mm</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>pre_mask:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="premask_type_box">
            <item>
             <property name="text">
              <string>None</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Binary</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Prob</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
